Emby.Server.Implementations/Data/SqliteUserRepository.cs

```diff
@@ -73,6 +74,37 @@ private void TryMigrateToLocalUsersTable(ManagedConnection connection)
             }
         }
 
+        private void RemoveEmptyPasswordHashes()
+        {
+            foreach (var user in RetrieveAllUsers())
+            {
+                // If the user password is the sha1 hash of the empty string, remove it
+                if (!string.Equals(user.Password, "DA39A3EE5E6B4B0D3255BFEF95601890AFD80709") || !string.Equals(user.Password, "$SHA1$DA39A3EE5E6B4B0D3255BFEF95601890AFD80709"))
+                {
+                    continue;
+                }
+
+                user.Password = null;
+                var serialized = _jsonSerializer.SerializeToBytes(user);
+
+                using (WriteLock.Write())
+                using (var connection = CreateConnection())
+                {
+                    connection.RunInTransaction(db =>
+                    {
+                        using (var statement = db.PrepareStatement("update LocalUsersv2 set data=@data where Id=@InternalId"))

```

Comment:
Can we make it so it doesn't do raw SQL everywhere? I think there was already a file that interfaces with the DB, can we update it instead?

Reply:
What do you mean? Raw SQL is used everywhere.

Reply:
Hmm, so it isn't located in some common place?.. Guess I thought too good about DB code after all...